### PR TITLE
Volleyballs

### PR DESCRIPTION
**Pull request by klorpa**
#### Summary

SUMMARY: Content "Adds volleyballs as items."
#### Purpose of change

Adds an indoor volleyball, beach volleyball, and rugby and volleyball variants of the jersey, including to item generation tables.

### CODE REVIEW

**Comment by cosmiccoincidence** [comment]
I gotta ask... did you add this because of #38155 ?

File: data/json/itemgroups/activities_hobbies.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
       [ "jersey", 30 ],
       [ "puck", 30 ],
       [ "basketball", 25 ],
+	  [ "indoor_volleyball", 20 ],
```
Comment (cosmiccoincidence) [inline data/json/itemgroups/activities_hobbies.json]: Indent.

File: data/json/itemgroups/gear_civilian.json [hunk 2]
```diff
@@ -109,6 +109,7 @@
       [ "novel_pulp", 16 ],
       [ "radio_car_box", 1 ],
       [ "basketball", 8 ],
+	  [ "beach_volleyball", 8 ],
```
Comment (cosmiccoincidence) [inline data/json/itemgroups/gear_civilian.json]: Indent.

File: data/json/items/armor/torso_clothes.json [hunk 3]
```diff
@@ -281,6 +281,14 @@
         "id": "monroeville",
         "text": "A hockey jersey made of thick material imprinted with the logo of the Monroeville Zombies."
       }
```
Comment (cosmiccoincidence) [inline data/json/items/armor/torso_clothes.json]: ,

File: data/json/items/armor/torso_clothes.json [hunk 4]
```diff
@@ -281,6 +281,14 @@
...
+      {
+        "id": "rugby",
+        "text": "A rugby jersey made of thick material imprinted with the logo of the Canberra Drop Bears."
+      }
```
Comment (cosmiccoincidence) [inline data/json/items/armor/torso_clothes.json]: ,

File: data/json/items/generic/toys_and_sports.json [hunk 5]
```diff
@@ -167,6 +167,36 @@
...
     "to_hit": -1
   },
+  {
+    "type": "GENERIC",
+    "id": "indoor_volleyball",
+    "symbol": "*",
+    "color": "white",
+    "name": "volleyball",
+    "category": "other",
+    "description": "A standard regulation volleyball",
```
Comment (matskuman5) [inline data/json/items/generic/toys_and_sports.json]:
```suggestion
    "description": "A standard regulation volleyball.",
```

**Pull request merged**